test-cli.py

```diff
@@ -75,6 +75,26 @@ def current_worksheet():
     assert m is not None
     return m.group(1)
 
+def current_user():
+    """
+    Return the uuid and username of the current username in a tuple
+    Does so by parsing the output of `cl uinfo` which by default returns the info
+    of the current user
+    """
+    my_name = run_command([cl, 'uinfo'])
```

Comment:
This is kind of gross...can we add flags to `cl uinfo` to make it return userid or username (kind of how `cl info -f` works) isntead?  In general, it'd be nice to have `cl` commands that have a mode that outputs JSON instead of text, so actually it might be easier to do that.